### ISSUE TITLE
Add link to IETF status page

### ISSUE THREAD

**Issue by kesara** [ietf-tools/www#480]
We should advertise [IETF system status](https://status.ietf.org/) page in the footer of IETF website. Maybe on the IAB website too.

**Comment by ghwood** [comment]
Agree. It would also be nice to have a “report a problem” option, like the IETF Datatracker.

**Comment by holloway** [comment]
@ghwood what do you think of something like this?

IETF footer,

![Image](https://github.com/user-attachments/assets/6ebf69e7-3b1e-471e-83fc-872f193e20ee)

IAB footer

![Image](https://github.com/user-attachments/assets/e06029c3-a987-41e5-9fb4-279bdbebd486)

**Comment by ghwood** [comment]
For www.ietf.org the link would be better as part of the "link to other sites" level of items (see screenshot) rather than appearing as site-wide navigation element. I think the proposal works for www.iab.org on an abstraction level (though the "Contact Us" element seems to be the outlier), however I'd be interested to know what @cindymorgan thinks. : <img width="528" alt="Image" src="https://github.com/user-attachments/assets/b01aae5e-a2c0-4e65-ac03-1ae017700bd3" />

**Comment by cindymorgan** [comment]
I agree with Greg that the IAB version works, and that the IETF version would be better where Greg suggests. Thanks!

**Comment by holloway** [comment]
@ghwood @cindymorgan 

From just looking at the code right now it seems like they're both content managed in the CMS (i.e. no code change necessary) so just add it in the admin panel:

* https://www.ietf.org/admin/
* https://iab.org/admin/

In the code they're referred to as `FooterLinks` so there's probably a Wagtail snippet in each. I'd make the changem myself but I don't have a login. If you need help just let me know.
